notebooks/exploratory_data_analysis.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        query_dict[os.path.basename(query_file_path).split('.')[0]] = query_file_path
</pre><p>I recomand to use pathlib instead of os.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/ted-data-eu/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Tanks for recommendation, change is done.

```python
query_dict[query_file_path.stem] = str(query_file_path)
```